src/csharp/Microsoft.Spark/Sql/ArrowArrayHelpers.cs

```diff
@@ -38,10 +39,61 @@ internal static class ArrowArrayHelpers
             ArrowTypeId.Binary,
         };
 
-        public static IArrowArray CreateEmptyArray<T>()
+        public static DataFrameColumn CreateEmptyColumn<T>()
         {
-            ArrayData data = BuildEmptyArrayDataFromArrayType<T>();
-            return ArrowArrayFactory.BuildArray(data);
+            DataFrameColumn ret;
+            if (typeof(T) == typeof(PrimitiveDataFrameColumn<bool>))
+            {
+                ret = new PrimitiveDataFrameColumn<bool>("Empty");
```

Comment:
You mean you'd like to see `return PrimitiveDataFrameColumn<bool>("Empty")` directly here instead of setting `ret` and returning it at the end of the method? I'm not opposed to it, but it doesn't change the behavior right?